specs/src/specs/state_machine_modules.md

```diff
@@ -9,7 +9,7 @@ Celestia app is built using the cosmos-sdk, and follows standard cosmos-sdk modu
 - [minfee](https://github.com/celestiaorg/celestia-app/blob/main/x/minfee/README.md)
 - [mint](https://github.com/celestiaorg/celestia-app/blob/main/x/mint/README.md)
 - [paramfilter](https://github.com/celestiaorg/celestia-app/blob/main/x/paramfilter/README.md)
-- [upgrade](https://github.com/celestiaorg/celestia-app/blob/main/x/upgrade/README.md)
+- [signal](https://github.com/celestiaorg/celestia-app/blob/main/x/signal/README.md)
```

Comment:
FWIW this was fixed in this other PR: https://github.com/celestiaorg/celestia-app/pull/3277/files#diff-486d4b34ae4f4466efeb5b1fbb390a4233df75a84e6bbc1948277edd2e2b3b4eL12

I just can't get two approvals on it 😭